draft-ietf-webtrans-overview.md

```diff
@@ -266,6 +266,12 @@ terminate a session
   The error code and string are optional; the default values are 0 and "".  The
   delivery of the error code and string MAY be best-effort.
 
+drain a session
+: Indicate to the peer that it expects the session to be gracefully terminated
+  as soon as possible.  Either endpoint MAY continue using the session and MAY
+  open new streams.  This signal is intended to allow middleboxes and endpoints
```

Comment:
s/middlebox/intermediary/

Reply:
Updated in 918de8392daa3df3a1547bca9f49fb87d58c3eda